recipe/meta.yaml

```diff
@@ -17,26 +17,33 @@ build:
 
 requirements:
   build:
-    - python
+    - python >=2.7
```

Comment:
`pandas-profiling` v2.x.y supports Python 3 and dropped Python 2 support

(more details https://python3statement.org/)

Reply:
Fixed. @sbrugman would you like to be listed as a recipe maintainer?

Reply:
Yes, thank you.